easybuild/framework/easyconfig/easyconfig.py

```diff
@@ -1226,6 +1226,31 @@ def get_parsed_multi_deps(self):
 
         return multi_deps
 
+    def find_dep_version_match(self, dep):
+        """Identify the correct version for this system from the choices provided. This returns the version to use."""
+        if isinstance(dep['version'], string_type):
+            self.log.debug("Version is already a string ('%s'), OK", dep['version'])
+            return dep['version']
+        elif dep['version'] is None:
+            self.log.debug("Version is None, OK", dep)
```

Comment:
the `, dep` shouldn't be there, unless you want to print the whole `dep` dict:

So, either:

```python
self.log.debug("Version is None, OK")
```

or

```python
self.log.debug("Version is None, OK (dep: %s)", dep)
```

Reply:
Done